src/components/UserViewLayout/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from "react";
+import { Header } from "../Header";
+import { BottomNav } from "../BottomNav";
+import styles from "./UserViewLayout.module.css";
+import { useHistory } from "react-router-dom";
+
+interface UserViewLayoutProps {
+  showHeader: boolean,
+  showBottomNav: boolean
+}
```

Comment:
Since these are for show or hide, you can set them as, 
```tsx
{
 showHeader?: boolean,
  showBottomNav?: boolean
}
```

so we don't need to pass props as `showHeader={true}` just
```tsx
<UserViewLayout showHeader /> 
...
// If you don't want to show it, just remove it
<UserViewLayout /> 
```